cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ConstraintsTest.scala

```diff
@@ -83,9 +85,10 @@ class ConstraintsTest extends DocumentingTestBase with SoftReset {
       title = "Create a unique constraint with specified index provider and configuration",
       text =
         s"""To create a unique constraint with a specific index provider and configuration for the backing index, the `OPTIONS` clause is used.
-          |Valid values for the index provider is `$nativeProvider` and `$nativeLuceneProvider`, default if nothing is specified is `$nativeProvider`.
-          |Valid configuration settings are `$cartesianMin`, `$cartesianMax`, `$cartesian3dMin`, `$cartesian3dMax`,
-          |`$wgsMin`, `$wgsMax`, `$wgs3dMin`, and `$wgs3dMax`.
+          |Valid values for the index provider is `$nativeProvider`, `$nativeLuceneProvider`, and `$rangeProvider`, default if nothing is specified is `$nativeProvider`.
+          |The index type of the backing index is set depending on the provider, the `$rangeProvider` generates a range index while the other providers generates a b-tree index.
+          |Valid b-tree configuration settings are `$cartesianMin`, `$cartesianMax`, `$cartesian3dMin`, `$cartesian3dMax`,
+          |`$wgsMin`, `$wgsMax`, `$wgs3dMin`, and `$wgs3dMax`. The range index have no configuration settings.
```

Comment:
I think this explanation is really good, however it is a bit of a shame that you have to look in the examples to get it. Wouldn't it be helpful to also have it [here](https://neo4j.com/docs/cypher-manual/4.4-preview/constraints/syntax/#administration-constraints-syntax-create), where it now only says `For constraints that are backed by an index, the index provider and configuration for the backing index can be specified using the OPTIONS clause.`?

Reply:
for the constraints that might be good yes (feels like it is harder for indexes)